estudios_socioeconomicos/load.py

```diff
@@ -0,0 +1,73 @@
+import pickle
+
+from estudios_socioeconomicos.models import Seccion, Subseccion, Pregunta, OpcionRespuesta
+
+
+def parse(name):
+    """ utility script to parse the study.
+    """
+    preguntas = {}
+    while True:
+        seccion = input('Ingrese el nombre de la seccion: ')
+        if seccion == 'n':
+            break
+        if seccion not in preguntas:
+            preguntas[seccion] = {}
+        while True:
+            subseccion = input('Ingrese el nombre de la subseccion dentro de %s: ' % seccion)
+            if subseccion == 'n':
+                break
+            curr = 1
+            if subseccion not in preguntas[seccion]:
+                preguntas[seccion][subseccion] = []
+            while True:
+                p = input('Ingrese el nombre de la pregunta: ')
+                if p == 'n':
+                    break
+                opt = input('Respuestas: ')
+                opt = opt.split(',')
+                rel_integrante = input('related? (y/n): ')
+                preguntas[seccion][subseccion].append({
+                    'texto': p,
+                    'numero': curr,
+                    'opciones': list(map(lambda x: x.strip(), opt)) if len(opt) > 1 else [],
+                    'relacionado_a_integrante': rel_integrante == 'y'
+                    })
+                curr += 1
+    print(preguntas)
+    pickle.dump(preguntas, open(name, 'wb'))
+
+
+def load_data(name='estudios_socioeconomicos.pkl'):
```

Comment:
The .pkl file in your commit is called "estudios_socioeconomicos.pkl" and if run from the manage.py I think it should include the path to where its saved.